modules/pel/peltool/peltool.py

```diff
@@ -461,6 +465,63 @@ def parsePelFromBmcID(path: str, bmcID: str, config: Config) -> None:
         print("PEL not found")
 
 
+def parsePelFromPLID(path: str, config: Config):
+    """
+    Parse and display PELs matching with the input PLID from the specified directory.
+    Returns: None
+    Prints a JSON-formatted string containing valid PELs in the specified directory.
+    """
+    plid = processId(config.plid)
+    root, file_list = getFileList(path, config.extension, config.rev)
+    final_summary = {}
+    for file in file_list:
+        with open(os.path.join(root, file), 'rb') as fd:
+            data = fd.read()
+            stream = DataStream(data, byte_order='big', is_signed=False)
+            try:
+                eid, summary = parsePELSummary(stream, config)
+                if eid :
+                    if plid in summary['PLID']:
+                        if config.hex:
+                            printPELInHexFormat(data)
+                        else:
+                            final_summary[eid] = summary
+            except Exception as e:
+                print(f"Exception: No PEL parsed for {file}: {e}")
+    if not config.hex:
+        print(prettyPrint(json.dumps(final_summary, indent=4) , desiredSpace = 29))
+
+
+def parsePelFromSRCID(path: str, config: Config):
+    """
+    Parse and display PELs matching with the input SRC ID from the specified directory.
+    Returns: None
+    Prints a JSON-formatted string containing valid PELs in the specified directory.
+    """
+    if config.srcid:
+        if len(config.srcid) > 32:
+            sys.exit('Invalid SRC ID length is provided !')
+
+    root, file_list = getFileList(path, config.extension, config.rev)
+    final_summary = {}
+    for file in file_list:
+        with open(os.path.join(root, file), 'rb') as fd:
+            data = fd.read()
+            stream = DataStream(data, byte_order='big', is_signed=False)
+            try:
+                eid, summary = parsePELSummary(stream, config)
+                if eid :
+                    if config.srcid and config.srcid in summary['SRC']:
```

Comment:
> config.srcid

is it needed? we will call only if the src id is provide in command-line, right?